app/src/main/java/com/github/neighbortrader/foodboardapp/handler/clientmodelHandler/UserHandler.java

```diff
@@ -44,11 +42,11 @@ public static User generateRandomUser() {
     }
 
     public static void saveToSharedPreferences(User user) {
-        Log.d(TAG, "saveToSharedPreferences()");
+        Log.d(TAG, "saveCurrentGroceriesToSharedPreferences()");
```

Comment:
was sind sharedPreferences nochmal ?

Reply:
http://letmegooglethat.com/?q=android+sharedpreferences

Reply:
also du speicherst den User in ein File ?